kip-0017.md

```diff
@@ -98,7 +98,7 @@ This contains information on which of the requested items it supports.
       "accounts": [
         "kadena:mainnet01:38298612cc2d5e841a232bd08413aa5304f9ef3251575ee182345abc3807dd89",
         "kadena:testnet04:38298612cc2d5e841a232bd08413aa5304f9ef3251575ee182345abc3807dd89",
-        "kadena:testnet04:22ddc64851718e9d41d98b0f33d5e328ae5bbbbd97aed9885adac0f2d070ff9c"
+        "kadena:development:22ddc64851718e9d41d98b0f33d5e328ae5bbbbd97aed9885adac0f2d070ff9c"
```

Comment:
Thank you for the callout!

The WC docs for [`Approving a session response`](https://docs.walletconnect.com/2.0/specs/clients/sign/namespaces#approving-a-session-response) state:
> The following is validated for the WalletConnect library:
> ...
> validates accounts are included at least one per chain in the requiredNamespaces

In the proposal request we ask for access to `"chains": ["kadena:mainnet01", "kadena:testnet04", "kadena:development"]`, so the WC library expects that at least one account per requested chain be included in `requiredNamespaces`.

Reply:
Correct. The previous structure would have thrown an error on the wallet side when it validated the `requiredNamespaces` with the namespaces provided.